pkgs/development/python-modules/dvc-task/default.nix

```
@@ -0,0 +1,68 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, setuptools-scm
+
+, kombu
+, shortuuid
+, celery
+, funcy
+, pytest-celery
+, pytest-mock
+, pytest-test-utils
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "dvc-task";
+  version = "0.1.0";
+  format = "pyproject";
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "iterative";
+    repo = pname;
+    rev = version;
+    hash = "sha256-g9R6A70wri1XaMnQM31H/dk9MpmQ3d1oNNScrIpXt4s=";
+  };
+
+  SETUPTOOLS_SCM_PRETEND_VERSION = version;
+
+  nativeBuildInputs = [
+    setuptools-scm
+  ];
+
+  propagatedBuildInputs = [
+    kombu
+    shortuuid
+    celery
+    funcy
+  ];
+
+  postPatch = ''
+    substituteInPlace setup.cfg \
+      --replace "kombu>=5.3.0a1" "kombu==5.2.4"
```

just to make this a bit less prone to failur if kombu does get updated
```suggestion
      --replace "kombu>=5.3.0a1" "kombu~=5.2"
```